docs/cudf/source/api_docs/general_functions.rst

```diff
@@ -14,6 +14,7 @@ Data manipulations
    cudf.cut
    cudf.factorize
    cudf.get_dummies
+   cudf.isclose
```

Comment:
Removing. We plan to deprecate this function and adding docs would signal that it is supported.
```suggestion
```